### ISSUE TITLE
Release version tags issue

### ISSUE THREAD

**Issue by nickg-ifit** [firebase/firebase-ios-sdk#9073]
Please forgive me for deleting the template.  This issue is related to the Github repository, but it is not related to any of the code.

As of version `v8.9.0` of this repository, the version tags started getting a `v` prefix, whereas up until `8.8.0`, it was strictly the version number.

This is preventing consumers from updating past `8.8.0` if when pulling the sdk in via Swift Package Manager, which does not accept the tag with the v prefix.

Can the `v8.9.0`, `v8.9.1`, and `v8.10.0` commits have a `8.9.0`, `8.9.1`, and `8.10.0` tags respectively appended, as well as using this format going forward?

Relevant Apple documentation: https://developer.apple.com/documentation/swift_packages/version

**Comment by ryanwilson** [comment]
#8683 for context on when we switched to `v` versions.

**Comment by nickg-ifit** [comment]
We are running into this on our automated build system using command line package resolve.

```
[20:43:04]: ▸ Command line invocation:
[20:43:04]: ▸     /Applications/Xcode-12.4.app/Contents/Developer/usr/bin/xcodebuild -resolvePackageDependencies -workspace CrashReporter.xcworkspace -scheme App -derivedDataPath ./DerivedData
[20:43:04]: ▸ User defaults from command line:
[20:43:04]: ▸     IDEDerivedDataPathOverride = /Users/vagrant/git/DerivedData
[20:43:06]: ▸ Resolve Package Graph
[20:43:13]: ▸ Resolved source packages:
[20:43:13]: ▸   git: /Users/vagrant/git
[20:43:13]: ▸ xcodebuild: error: Could not resolve package dependencies:
[20:43:13]: ▸   Invalid semantic version string 'v8.10.0'
```

**Comment by paulb777** [comment]
Thanks, looks a difference between Xcode 12 and Xcode 13.  I'll add tags to the repo and update our release process for at least the few remaining months that we continue to support Xcode 12.

**Comment by nickg-ifit** [comment]
![](https://files.slack.com/files-pri/T020F1LHZ5K-F02Q44707EZ/image.png)

We are also getting this on Xcode 13.1.

**Comment by paulb777** [comment]
Weird, we don't reproduce with Xcode 13.  Is the issue resolved for you now that the 8.10.0 tag exists on the repo?

**Comment by nickg-ifit** [comment]
Yes.  The `8.10.0` tag without the `v` prefix resolves the issue.  Thanks for the quick response.
